src/CMakeLists.txt

```diff
@@ -3,25 +3,42 @@ cmake_minimum_required(VERSION 3.10) # Set the minimum required version of CMake
 project(ExperimentalJsonBuilder) # Name your project
```

Comment:
Not sure that `project` is needed unless you expect someone to take the src directory and use that as a distinct project.

Reply:
Will try to delete it in follow-up PR and see if it has any impact.